subsys/nrf_security/src/core/lite/psa_core_lite.c

```diff
@@ -215,6 +215,40 @@ psa_status_t psa_hash_compute(
 					       hash, hash_size, hash_length);
 }
 
+psa_status_t psa_hash_setup(psa_hash_operation_t *operation,
+                            psa_algorithm_t alg)
+{
+	if (operation == NULL) {
+		return PSA_ERROR_INVALID_ARGUMENT;
+	}
+
+	return psa_driver_wrapper_hash_setup(operation, alg);
+}
+
+psa_status_t psa_hash_update(psa_hash_operation_t *operation,
+                             const uint8_t *input,
+                             size_t input_length)
+{
+
+	if (operation == NULL || input == NULL) {
+		return PSA_ERROR_INVALID_ARGUMENT;
+	}
+
+	return psa_driver_wrapper_hash_update(operation, input, input_length);
+}
+
+psa_status_t psa_hash_finish(psa_hash_operation_t *operation,
+                             uint8_t *hash,
+                             size_t hash_size,
+                             size_t *hash_length)
+{
+	if (operation ==  NULL || hash == NULL || hash_length == NULL) {
+		return PSA_ERROR_INVALID_ARGUMENT;
+	}
+
+	return psa_driver_wrapper_hash_finish(operation, hash, hash_size, hash_length);
+}
+
```

Comment:
Why are you adding in three functions that do the same thing as three already existing functions in this file?

Reply:
My mistake.

This was ported from a revision of LITE that didn't have them, and I didn't notice they were here in this revision.